.travis.yml

```
@@ -0,0 +1,12 @@
+language: python
+matrix:
```

You don't really need a matrix. The task was to have just Python 3.7 there.
But if you want both, there's matrix expansion for certain options like:
```suggestion
python:
- "3.7"
- "3.6"
```